frontend/src/constants/langEnUs.js

```diff
@@ -10,6 +10,8 @@ export const BTN_DELETE = 'Delete';
 export const BTN_DELETE_DRAFT = 'Delete Draft';
 export const BTN_EDIT_DRAFT = 'Edit Draft';
 export const BTN_EDIT_PVR_DRAFT = 'Edit PVR';
+export const BTN_NEW_TRANSFER = 'New Transfer';
+export const BTN_NEW_CREDIT_TRANSACTION = 'New Credit Transaction';
```

Comment:
Added 2 strings that weren't in the Lang file, while I was already there adding IDs.